server/config/config.js

```diff
@@ -4,7 +4,7 @@ dotenv.config();
 
 module.exports = {
   development: {
-    url: process.env.DATABASE_URL,
```

Comment:
Hi Peace,

Great Job! 

Was this change reflected in the sampleenv file?

Reply:
Yes thank Princess